src/ui/localization.ts

```diff
@@ -140,6 +140,138 @@ const localizeInit = () => {
+
+                        // Shortcuts panel
+                        "shortcuts.title": "RACCOURCIS CLAVIERS",
+                        "shortcuts.tools": "OUTILS",
+                        "shortcuts.move": "Déplacer",
+                        "shortcuts.rotate": "Tourner",
+                        "shortcuts.scale": "Changer l'échelle",
+                        "shortcuts.rect-selection": "Sélection avec rectangle",
+                        "shortcuts.brush-selection": "Sélection avec pinceau",
+                        "shortcuts.picker-selection": "Sélection avec pipette",
+                        "shortcuts.brush-size": "Augmenter/Diminuer la taille du pinceau",
+                        "shortcuts.deactivate-tool": "Désactiver l'outil",
+                        "shortcuts.selection": "SELECTION",
+                        "shortcuts.select-all": "Tout sélectionner",
+                        "shortcuts.deselect-all": "Tout desélectionner",
+                        "shortcuts.invert-selection": "Inverser la sélection",
+                        "shortcuts.add-to-selection": "Ajouter à la sélection",
+                        "shortcuts.remove-from-selection": "Retirer de la sélection",
+                        "shortcuts.delete-selected-splats": "Supprimer splats sélectionnés",
+                        "shortcuts.show": "AFFICHER",
+                        "shortcuts.hide-selected-splats": "Masquer splats sélectionnés",
+                        "shortcuts.unhide-all-splats": "Réafficher tous les splats",
+                        "shortcuts.toggle-data-panel": "Afficher/Cacher l'onglet données",
+                        "shortcuts.other": "AUTRES",
+                        "shortcuts.select-next-splat": "Sélectionner le splat suivant",
+                        "shortcuts.undo": "Annuler",
+                        "shortcuts.redo": "Rétablir",
+                        "shortcuts.toggle-splat-overlay": "Basculer affichage splat",
+                        "shortcuts.focus-camera": "Focaliser la caméra sur la sélection actuelle",
+                        "shortcuts.toggle-camera-mode": "Basculer le mode de camera",
+                        "shortcuts.toggle-grid": "Afficher/Cacher la grille",
+                        "shortcuts.toggle-gizmo-coordinate-space": "Basculer en espace de coordonnées Gizmo",
+
+                        // popup
+                        "popup.ok": "OK",
+                        "popup.cancel": "Annuler",
+                        "popup.yes": "Oui",
+                        "popup.no": "Non",
+                        "popup.error-loading": "Erreur de chargement du fichier",
+
+                        // right toolbar
+                        "tooltip.splat-mode": "Mode splat ( M )",
+                        "tooltip.show-hide": "Afficher/cacher les splats ( Barre espace )",
+                        "tooltip.frame-selection": "Sélection de l'image ( F )",
+                        "tooltip.view-options": "Afficher les options",
```

Comment:
Again, I think this means "Show options" with 'show' being a verb. Here, 'View' is a noun, meaning the 3D viewport.